src/main/java/edu/ucsb/cs156/rec/entities/RecommendationRequest.java

```diff
@@ -25,6 +27,10 @@ public class RecommendationRequest {
   @GeneratedValue(strategy = GenerationType.IDENTITY)
   private long id;
 
+  @ManyToOne
+  @JoinColumn(name = "user_id", referencedColumnName = "id", insertable = false, updatable = false)
+  private User user;
+
```

Comment:
See the considerable changes to the database in the code review for pr #66 ... this will need to be updated based on the final resolution of pr #66